node/actors/bft/src/leader/state_machine.rs

```diff
@@ -46,10 +48,19 @@ pub(crate) struct StateMachine {
 impl StateMachine {
     /// Creates a new StateMachine struct.
```

Comment:
document how to use the result, which is nontrivial now.

Reply:
Fixed, changes here: https://github.com/matter-labs/era-consensus/pull/56/commits/500b17f0ba75d3edebc77a04ae04d7de5a47fc5a